components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/ext/ExtendedAPI.java

```diff
@@ -30,6 +32,8 @@
     private String mgwApiSecurity = null;
     //Scopes
     private String mgwApiScope = null;
+    //support apim application level security
+    private List<String> apiSecurityByExtension = null;
```

Comment:
Rename variable
Suggestion: `extensionSecurity`

Reply:
Renamed to applicationSecurity